jekyll/_cci2/deployment-integrations.md

```diff
@@ -507,6 +507,46 @@ If you would like to simplify your configuration workflows using a CircleCI orb
 
 For more detailed information about this orb, refer to the [CircleCI Google Cloud Orbs](https://circleci.com/orbs/registry/orb/circleci/gcp-cli) page in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
 
+## Hapistrano
+
+[Hapistrano](https://github.com/stackbuilders/hapistrano) is a deployment library based on Haskell similar to Ruby's Capistrano. Hapistrano (like Capistrano for Ruby) deploys applications to a new directory marked with a timestamp on the remote host. It creates this new directory quickly by placing a git repository for caching purposes on the remote server. Hapistrano uses a yml deploy file to perform tasks on the remote servers.
+
+```yaml
+
+version: 2
+jobs:
+  deploy:
+    docker:
+      - image: stackbuilders/hapistrano:v0.3.9.2
+
+    steps:
+      - checkout
+      - deploy:
+          name: Deploy
+          command:
+            hap deploy -c deploy/deploy-file.yaml
```

Comment:
@nickovivar according to Hapistrano README.md the configuration file is usually named `hap.yaml`:

```diff
- hap deploy -c deploy/deploy-file.yaml
+ hap deploy -c hap.yaml
```